### ISSUE TITLE
After moving show folder, .nfo files show up in old location after refresh

### ISSUE THREAD

**Issue by GregSawin** [SickRage/sickrage-issues#416]
I moved the location of where my shows are stored via the mass update feature but nfo files keep showing up in the old location.  Looks like this happens after a refresh or clicking the 'Re-scan files' button on a particular show.  The database holds the original location per episode so I'd guess it's trying to put the .nfo in that folder which no longer exists but the process creates it anyway.  Perhaps after a mass update the episode location could be updated to this location as well.  I appear to have resolved my issue by manually updating the database via: update tv_episodes set location=replace(location,[old root], [new root])

Steps to reproduce:
1) Manually (outside of SR) created a copy of my shows folder in the new location
2) Within SR using mass update I changed the 'Root directories' to the new location
3) Later on I deleted the old show folders
4) Days later I noticed the old location has reappeared with some of the shows, the only thing in these folders is the per episode .nfo files.  You can force this by hitting 'Re-Scan files' on any show that had previously downloaded episodes to the old location.

I'm running this on Win10 x64.
Branch: master
Commit: 4eccee2ad1b3e3d144bfd8e315fdf397cfd33361
Version: v5.1

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist)

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by fernandog** [comment]
@GregSawin maybe the mass change location didn't worked

**Comment by GregSawin** [comment]
No, it worked.  Editing any of the shows afterwards shows the new folder and new shows appear in the new folder.

**Comment by fernandog** [comment]
@GregSawin so you are saying the show folder is changed but episodes in tv_episodes root are not ?

**Comment by GregSawin** [comment]
Afterwards I manually moved anything from the old to the new.  The issue is that new .nfo files keep appearing in the old location afterwards.

**Comment by fernandog** [comment]
@SickRage/moderators 
SR must change location of episodes in tv_episodes too if location is not null
